ecs/spotfleet/cloud-config.yml

```diff
@@ -0,0 +1,80 @@
+#cloud-config
+package_update: true
```

Comment:
Can't you just use `template_file` and a relative path (`../cluster/cloud-config.yml`)?

Reply:
No - I tried if you call it from outside the repo - then it seems terraform init does not copy files outside the module you have asked for
```
Error: module.this-poc.module.cluster.data.template_file.main: 1 error(s) occurred:

* module.this-poc.module.cluster.data.template_file.main: file: open ../../cluster/cloud-config.yml: no such file or directory in:

${file("../../cluster/cloud-config.yml")}
```

Reply:
Ah, of course - my bad!